packages/loaders/code-file/src/index.ts

```diff
@@ -58,6 +67,11 @@ export class CodeFileLoader implements UniversalLoader<CodeFileLoaderOptions> {
     pointer: SchemaPointerSingle | DocumentPointerSingle,
     options: CodeFileLoaderOptions
   ): Promise<boolean> {
+    if (isGlob(pointer)) {
+      // FIXME: parse to find and check the file extensions?
```

Comment:
Oh, maybe this is a non-issue: the graphql-file and code-file loaders can both resolve globs for the other because they're both on the filesystem? So as long as there isn't a protocol prefix like `git:` in front (or is a `file:` protocol though that might be a separate feature PR) blindly returning true for all (local file system) globs is fine?

Reply:
@PixnBits It is kind of tricky but let's try that and see how the tests will be.